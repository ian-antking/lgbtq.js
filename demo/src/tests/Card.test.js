import React from 'react';
import { render } from '@testing-library/react';
import Card from '../components/Card';

describe('Card', () => {
  it('renders the correct snapshot', () => {
    const { asFragment } = render(<Card />);
    
    expect(asFragment()).toMatchSnapshot();
  })

  it('renders the correct background', () => {
    const { getByTestId } = render(<Card data-testid="test-card" background="red" />);

    expect(getByTestId('test-card')).toHaveStyle('background: red');
  })
})