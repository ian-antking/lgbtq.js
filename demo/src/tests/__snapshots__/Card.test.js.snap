// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders the correct snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-AxjAm iDBQeS"
  />
</DocumentFragment>
`;
